{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["poetry==0.12.11"]
+requires = ["poetry==0.12.12"]
```

Comment:
Also: https://wemake-django-template.readthedocs.io/en/latest/pages/template/development.html#updating-poetry-version

And: https://github.com/wemake-services/wemake-django-template/blob/master/pyproject.toml#L2

Reply:
Nice catch.

Reply:
One more place: https://github.com/wemake-services/wemake-django-template/blob/master/pyproject.toml#L2

Sorry for the complex process.

Reply:
Force pushed update, also updated at travis file